### PR TITLE
Root to leaf sum

### PR DESCRIPTION
**Pull request by Aarti002**
issue #917 solved
python

### CODE REVIEW

File: Tree/Python/root_to_leaf_sum.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        if(root.left!=None):
+            res= res or path_sum(root.left,rem)
+        if(root.right!=None):
+            res= res or path_sum(root.right,rem)
+        return res
+
+
+if __name__=='__main__':
+    root=new_node(11)
+    root.left=new_node(12)
```
Comment (plazzy99) [inline Tree/Python/root_to_leaf_sum.py:53]: update the code to take dynamic input

File: Tree/Python/root_to_leaf_sum.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    root.right=new_node(21)
+    root.left.right=new_node(5)
+    root.left.left=new_node(30)
+    root.right.right = new_node(15)
+    root.right.left = new_node(3)
+    sum=47
+    print(path_sum(root,sum))
+    
+#output: True
```
Comment (plazzy99) [inline Tree/Python/root_to_leaf_sum.py:62]: 2 test cases with time and space complexity

File: Tree/Python/root_to_leaf_sum.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+#Ques: Check weather it is possible to get sum from root node to a leaf node
+#language: python
```
Comment (plazzy99) [inline Tree/Python/root_to_leaf_sum.py:2]: describe the question little bit

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Update the readme

**Comment by Aarti002** [comment]
hey @plazzy99 i have added examples and  complexities please review it once.
Thank u!

**Comment by plazzy99** [comment]
> hey @plazzy99 i have added examples and complexities please review it once.
> Thank u!

update readme

**Comment by Aarti002** [comment]
please check once i have added my code link in readme file @plazzy99 

Thank you!

**Comment by Aarti002** [comment]
@plazzy99 please review my code again
Thank you!

File: Tree/readme.md [hunk 4]
```diff
@@ -48,22 +48,23 @@ that, this property should be satisfied at every node in the tree.
...
 * Lowest common ancestor in a binary tree ----> [Python](/Code/Python/LCA_in_binary_tree.py)
+* Maiximum  sum path from any node to any node in Binary Tree  ----> [C++](/Code/C++/max_tree_path.cpp)
+* Maximum Topology Short ----> [C++](/Code/C++/Max_Topology_Short.cpp)
+* N-ary Tree ----> [Python](/Code/Python/n_ary_tree.py)
 * Red Black Tree (Insertion and Deletion operation) ----> [Java](/Code/Java/RedBlackTree.java) 
 * Right Side View of Binary Tree ----> [C++](/Code/C++/Right_Side_View_of_Binary_Tree.cpp)
 * Root to Leaf Path with Given Sum ----> [C++](/Code/C++/Root_to_leaf_path_with_given_sum.cpp)
+* Root to Leaf Sum ----> [Python](/Code/Python/root_to_leaf_sum.py)
+* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp) 
+* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp)
```
Comment (plazzy99) [inline Tree/readme.md:66]: duplicate bst added please correct this

**Review by plazzy99 — CHANGES_REQUESTED** [review]

**Comment by Aarti002** [comment]
@plazzy99 i have made requested changes please check once.
Thank you!

**Comment by tarun26091999** [comment]
@Aarti002 conflict present.

**Comment by Aarti002** [comment]
sorry for conflicts you can review now @tarun26091999 
Thank you!

**Pull request merged**